.changeset/pretty-garlics-stare.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
I don't think it makes sense to bump versions on any packages since no code is changing.